src/EditModel/event-program/reducers.js

```diff
@@ -3,35 +3,59 @@ import {
     EVENT_PROGRAM_STEP_CHANGE,
     EVENT_PROGRAM_STEP_NEXT,
     EVENT_PROGRAM_STEP_PREVIOUS,
+    EVENT_PROGRAM_LOAD_SUCCESS,
+    EVENT_PROGRAM_LOAD_ERROR,
+    PROGRAM_STEPPER_SET_DISABLE,
 } from './actions';
 import { STEPPER_RESET_ACTIVE_STEP } from '../actions';
 import { stageNotificationsReducer } from './notifications/reducers';
+import { programStageStepperReducer} from "./tracker-program/program-stages/reducer";
 import steps from './event-program-steps';
 import { next, previous, first } from '../stepper/stepIterator';
 
-function eventProgramStepperReducer(state = { activeStep: first(steps) }, action) {
+function eventProgramStepperReducer(
+    state = { activeStep: first(steps), disabled: false, isLoading: true },
+    action
+) {
     switch (action.type) {
-    case EVENT_PROGRAM_STEP_CHANGE:
-        return {
-            activeStep: action.payload,
-        };
+        case EVENT_PROGRAM_STEP_CHANGE:
+          //  const disabled = action.payload == ""
```

Comment:
I see that this is commented out, but another trivia question.

If this was not commented, in what scope would the `const disabled` declared?
What are the effects on the surrounding code?
Are you able to reuse the `disabled` variable elsewhere?
Can you mitigate the issues?

(See this for the explanation: https://eslint.org/docs/rules/no-case-declarations)